src/collection-view.js

```diff
@@ -711,9 +712,14 @@ const CollectionView = Backbone.View.extend({
     return false;
   },
 
+  // Override to handle DOM inserting differently
+  beforeEl(el, siblings) {
+    this.Dom.getEl(el).before(siblings);
```

Comment:
So you would prefer that `beforeEl` go back on to the DOM API for the purpose of supporting the older `CollectionView`?